webui/react/src/hooks/useFeature.ts

```diff
@@ -1,25 +1,32 @@
 import queryString from 'query-string';
+import { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
 
 import { useStore } from 'contexts/Store';
 
 interface FeatureHook {
-  isOnRbac: () => boolean
-}
-
-interface Queries {
-  rbac?: string;
+  isOn: (ValidFeature: string) => boolean
 }
 
 const useFeature = (): FeatureHook => {
-  return { isOnRbac: () => IsOnRbac() };
+  const location = useLocation();
+  const [ features, setFeatures ] = useState<string[]>([]);
+  useEffect(() => {
+    const queries = queryString.parse(location.search);
```

Comment:
You should do this globally at the top of the file so it happens when the application loads instead of when the hook is used.